src/scanners/css.js

```diff
@@ -26,7 +26,15 @@ export default class CSSScanner extends BaseScanner {
                 }
 
                 this.validatorMessages = this.validatorMessages.concat(
-                  rules[cssRule](rule));
+                  rules[cssRule](
+                    rule.declarations,
```

Comment:
I think I'd rather pass the whole cssRule object and then the rule function has access to everything. For example we need access to selectors to be able to reproduce one of the existing tests see https://github.com/mozilla/amo-validator/search?utf8=%E2%9C%93&q=%23identity-box